capstone-app/src/app/modules/maps/map/map.component.html

```diff
@@ -1,5 +1,30 @@
-<google-map
-    [center] = "location"
-    height = "500px"
-    width = "100%"
-></google-map>
\ No newline at end of file
+<mat-form-field class="example-form-field">
+    <mat-label>Search for a city</mat-label>
+    <input matInput type="text"  #searchBar>
+    <button mat-button 
+    *ngIf="searchBar.value" 
+    matSuffix mat-icon-button aria-label="Clear" 
+    (click)="searchBar.value=''">
+      <mat-icon>close</mat-icon>
+    </button>
+</mat-form-field>
+<button mat-icon-button color="primary" (click)="searchButton(searchBar.value)" >
+    <mat-icon >search</mat-icon>
+</button>
+
+<div #mapContainer>
+    <google-map
+    [center]="getCurrentOrSetLocation()"
+    height="750px"
```

Comment:
Shouldn't height and width go in your CSS file?

Reply:
can't it won't work from css it has to in the google map tag